### PR TITLE
Add support for Flipkart

### PR DESCRIPTION
**Pull request by dylants**
Add support for the Flipkart site.  This implementation uses page scraping to obtain the product information.

### CODE REVIEW

File: lib/sites/flipkart.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    // process the price string in rupees
+    const price = siteUtils.processPrice(`R ${priceString}`);
+
+    return price;
+  }
+
+  findCategoryOnPage() {
+    // default to other
```
Comment (rajkumarpb) [inline lib/sites/flipkart.js]: Any issue in finding category? Any reason why you leave it to OTHER? Just curious.

**Comment by rajkumarpb** [comment]
Nice work @dylants. Guess we can do the same for snapdeal too!!

File: lib/sites/flipkart.js (hunk 1 above)
Comment (dylants) [inline lib/sites/flipkart.js]: I was just trying to get something up quickly to provide an example scraping implementation.  Please feel free to add to this branch to find the category.

Comment (rajkumarpb) [inline lib/sites/flipkart.js]: Ok. Just get this merge. Will add them up in a couple of days.

Comment (dylants) [inline lib/sites/flipkart.js]: Okay, I opened #69 to track that work.

**Pull request merged**